Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -820,11 +818,16 @@ private void HandleOrderEvent(OrderEvent fill)
             {
                 Log.Debug("BrokerageTransactionHandler.HandleOrderEvent(): " + fill);
                 Interlocked.Exchange(ref _lastFillTimeTicks, DateTime.Now.Ticks);
+
+                // set the price currency on the fill and on the order
+                var security = _algorithm.Securities[fill.Symbol];
+                fill.FillPriceCurrency = security.SymbolProperties.QuoteCurrency;
+                order.PriceCurrency = fill.FillPriceCurrency;
```

Comment:
I thought about this as well, but looking at the brokerage code, I thought making changes to all brokerages at once was a bit too risky.
Perhaps for now we could set the fill currency (in the above code) only if empty (i.e. not set by the brokerage). This would allow us to update and test the brokerages independently.
Thoughts ?